app/src/test/java/com/example/rickandmorty/LocationViewModelTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.rickandmorty
+
+class LocationViewModelTest {
```

Comment:
You should try to avoid committing commented code, I understand that probably this test will be done later, so it would better to committed when is ready to review. That way you can keep the repo cleaner and makes the PRs easier to review.